[Scout] Align skills and docs with current Scout behavior (#281056)
elastic/kibana: .agents/skills/cypress-to-scout-migration/references/migration-best-practices.md

- [Parallelization](#parallelization)
- [Fixtures](#fixtures)
- [Package Organization](#package-organization)

> - [EUI Component Objects](#eui-component-objects)

Reviewer: ```suggestion
- [EUI Test Helpers](#eui-test-helpers)
```

Put shared code in `@kbn/scout`, security-specific code in `@kbn/scout-security`.

> ## EUI Component Objects

Reviewer: ```suggestion
## EUI Test Helpers
```

For EUI components that are non-trivial to drive, prefer the published `@elastic/eui-test-helpers` Component Objects exposed pre-bound to the page through `page.components.*`:

```typescript
await page.components.comboBox('fieldSelectorComboBox').setSelectedOptions(['host.name']);
```

Available factories include `page.components.comboBox(testSubj)`, `page.components.dataGrid(testSubj)`, `page.components.superSelect(testSubj)`, and `page.components.globalToastList()`. For simple, native-like components such as text fields and checkboxes, use plain Playwright locators.

> Existing `Eui*Wrapper` classes exported by `@kbn/scout` are compatibility fallbacks only when no equivalent Component Object exists. Do not add or extend wrappers in a test suite.

Reviewer: ```suggestion
Use locators when no equivalent EUI test helpers exist. Do not add or extend component objects in a test suite.
```
